spec/lib/miq_automation_engine/miq_ae_service_model_spec.rb

```diff
@@ -136,4 +136,25 @@ module MiqAeServiceModelSpec
       end.to raise_error(NoMethodError)
     end
   end
+
+  context 'taggable' do
```

Comment:
@gmcculloug A more general question: do we need to have tests to make sure things we expect them to be taggable actually do?  
Adding this as a lightweight test setup that we can just add a newly taggable to L141. 
What do you think?